### PR TITLE
Additional Fields

### PR DESCRIPTION
**Pull request by hcjk**
Adds some of the additional fields we need to display

### CODE REVIEW

File: src/components/AuthStateButton.tsx [hunk 1]
```diff
@@ -14,14 +16,14 @@ export default function AuthStateButton(): JSX.Element {
   return (
     <>
       {!authState.loading && (
-        <>
+        <Box ml={3}>
```
Comment (jake-billings) [inline src/components/AuthStateButton.tsx]: I'm guessing this styles the sign in/sign out button too. Awesome!

File: src/views/CreateHospitalView.tsx [hunk 2]
```diff
@@ -20,13 +20,18 @@ type PublicProps = RouteComponentProps;
 const defaultHospital = {
   name: '',
   location: {
-    address: '',
+    address: 'asdf',
```
Comment (jake-billings) [inline src/views/CreateHospitalView.tsx]: Why is the default address `'asdf'` instead of `''`

File: src/views/EditHospitalView.tsx [hunk 3]
```diff
@@ -25,13 +25,18 @@ type PublicProps = RouteComponentProps & {
 const defaultHospital = {
   name: '',
   location: {
-    address: '',
+    address: 'asdf',
```
Comment (jake-billings) [inline src/views/EditHospitalView.tsx]: Same question

File: src/components/HospitalInput.tsx [hunk 4]
```diff
@@ -16,143 +18,199 @@ type State = {
...
-          googleMapsPlaceId: '',
-          lat: 0,
-          lng: 0,
-        },
-        totalBedCount: 0,
-        occupiedBedCount: 0,
-      },
-    };
-  }
+const defaultHospital = {
```
Comment (jake-billings) [inline src/components/HospitalInput.tsx]: Okay fine just delete the documentation

File: src/views/CreateHospitalView.tsx (hunk 2 above)
Comment (hcjk) [reply src/views/CreateHospitalView.tsx]: I'll drop that. Not sure where it came from.

File: src/components/HospitalInput.tsx (hunk 4 above)
Comment (hcjk) [reply src/components/HospitalInput.tsx]: lmao, sorry we can reorganize it

**Comment by jake-billings** [comment]
Actually, stuff is definitely broken now that I open the app in a browser

The edit hospital view doesn't load data anymore
Save buttons no longer have `cursor: pointer` when hovering
The textbox for number of COVID patients is labeled "Phone"

Let's reserve merge requests for versions of the app the have gone through some level of quality control

**Pull request merged**